bash/install_linux_packages.sh

```diff
@@ -4,4 +4,4 @@
 set -e
 
 apt-get update
-apt-get --assume-yes install --no-install-recommends postgresql-client gdal-bin
+apt-get --assume-yes install --no-install-recommends postgresql-client gdal-bin git-core bash-completion
```

Comment:
Added this in monorepo as well. Basically just so that you can type `git checkout` and hit tab to see list of branches, or autocomplete once you've typed part of the branch name